components/dialog/dialog-fullscreen.js

```diff
@@ -127,10 +127,6 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 					animation: d2l-dialog-fullscreen-open 400ms ease-out;
 				}
 
-				dialog::backdrop {
-					transition: opacity 200ms ease-out;
-				}
```

Comment:
This is removed because it's already captured in `dialog-styles.js`, now that both are `ease-out`.